### PR TITLE
Create invert-binary-tree.java

### PR DESCRIPTION
**Pull request by soyeon51696**
recursion is my weakness 🥲 i do always without knowing how it works. but it works. My brain is too lazy to understand recursion🫣

### CODE REVIEW

File: study1/week2/invert-binary-tree.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ *         this.val = val;
+ *         this.left = left;
+ *         this.right = right;
+ *     }
+ * }
+ */
+
+class Solution {
+    public TreeNode invertTree(TreeNode root) {
+        //time complexity: O(nlogn) bacause it is binary Tree archtecture
```
Comment (h-joo) [inline study1/week2/invert-binary-tree.java:21]: Can you try to think again? how many nodes are you visiting?

File: study1/week2/invert-binary-tree.java
Comment (h-joo) [inline study1/week2/invert-binary-tree.java:1]: What is it of recursion that you're confused about? I can explain in the class if you can elaborate.

**Review by h-joo — APPROVED** [review]

File: study1/week2/invert-binary-tree.java (hunk 1 above)
Comment (soyeon51696) [reply study1/week2/invert-binary-tree.java:21]: n nodes....? then is O(n) right?

File: study1/week2/invert-binary-tree.java
Comment (soyeon51696) [reply study1/week2/invert-binary-tree.java:1]: hmmm I'm not exactly sure what I don't know when I use recursion 😂
But I always confused about the return value 🤔

**Pull request merged**